astronomer/providers/amazon/aws/example_dags/example_s3.py

```diff
@@ -22,11 +22,13 @@
 S3_BUCKET_WILDCARD_KEY = os.environ.get("S3_BUCKET_WILDCARD_KEY", "test*")
 PREFIX = os.environ.get("S3_PREFIX", "test")
 INACTIVITY_PERIOD = float(os.environ.get("INACTIVITY_PERIOD", 5))
-REGION_NAME = os.environ.get("REGION_NAME", "us-east-2")
+AWS_DEFAULT_REGION = os.environ.get("AWS_DEFAULT_REGION", "us-east-1")
```

Comment:
Changed ENV name here since AWS API/CLI expect region ENV name `AWS_DEFAULT_REGION` so that it would inline with AWS